app/components/views/GovernancePage/Proposals/Details/helpers.js

```diff
@@ -93,9 +94,20 @@ export const TimeValue = ({ timestamp, tsDate }) => (
 
 // This changes links to never open. Debatable whether we want to
 // allow proposals to link somewhere directly from decrediton.
-const renderInternalProposalLink = ({ children }) => {
-  return <a onClick={() => {} } href="#">{children}</a>;
-};
+const renderInternalProposalLink = ({ children, href }) => (
+  <InvisibleConfirmModalButton
+    modalTitle={<T id="politeia.details.openLinkModal" m="Open Link in External Browser"/>}
+    modalContent={<>
+      <T id="politeia.details.openLinkModalDesc1"
+        m="Click Confirm to open the link:  " />
+      <span className="prop-details-modal-link">{href}</span>
+      <T id="politeia.details.openLinkDesc2" m="in an external browser." />
+    </>}
```

Comment:
Use a value with the the link in the translation, otherwise it gets harder to properly translate.